assessments/homework.sql

```diff
@@ -93,7 +94,8 @@ INSERT INTO question_score_logs
 UPDATE instance_questions AS iq
 SET
     points_in_grading = least(iq.points + iq.current_value, aq.max_points) - iq.points,
-    score_perc_in_grading = least(iq.points + iq.current_value, aq.max_points) / aq.max_points * 100 - iq.score_perc
+    score_perc_in_grading = least(iq.points + iq.current_value, aq.max_points)
```

Comment:
Actually I think it's correct as is. This calculation makes `score_perc_in_grading` equal to the newly updated value of `points_in_grading` divided by `max_points` (minus `score_perc`).